website/docs/reference/resource-configs/athena-configs.md

```diff
@@ -0,0 +1,604 @@
+  - `update_condition` (optional): SQL condition used to identify records that should be updated.
+  - `insert_condition` (optional): SQL condition used to identify records that should be inserted.
+    - `incremental_predicates`, `delete_condition`, `update_condition` and `insert_condition` can include any column of the incremental table (`src`) or the final table (`target`). Column names must be prefixed by either `src` or `target` to prevent a `Column is ambiguous` error.
+
+Example of `delete_condition`:
+
+```sql
+{{ config(
+    materialized='incremental',
+    table_type='iceberg',
+    incremental_strategy='merge',
+    unique_key='user_id',
+    incremental_predicates=["src.quantity > 1", "target.my_date >= now() - interval '4' year"],
+    delete_condition="src.status != 'active' and target.my_date < now() - interval '2' year",
+    format='parquet'
+) }}
+
+select 'A' as user_id,
+       'pi' as name,
+       'active' as status,
+       17.89 as cost,
+       1 as quantity,
+       100000000 as quantity_big,
+       current_date as my_date
+```
+
+`update_condition` example:
+
+```sql
+{{ config(
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key=['id'],
+        update_condition='target.id > 1',
+        schema='sandbox'
+    )
+}}
+
+{% if is_incremental() %}
+
+select * from (
+    values
+    (1, 'v1-updated')
+    , (2, 'v2-updated')
+) as t (id, value)
+
+{% else %}
+
+select * from (
+    values
+    (-1, 'v-1')
+    , (0, 'v0')
+    , (1, 'v1')
+    , (2, 'v2')
+) as t (id, value)
+
+{% endif %}
+```
+
+Example of `insert_condition`:
+
+```sql
+{{ config(
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key=['id'],
+        insert_condition='target.status != 0',
+        schema='sandbox'
+    )
+}}
+
+select * from (
+    values
+    (1, 0)
+    , (2, 1)
+) as t (id, status)
+
+```
+
+### High availablity table (HA)
+
+The current implementation of table materialization can lead to downtime, as the target table is dropped and re-created. To have less destructive behavior, it's possible to use the `ha` config on your `table` materialized models. It leverages the table versions feature of the glue catalog, creating a temp table and swapping the target table to the location of the temp table. This materialization is only available for `table_type=hive` and requires using unique locations. For Iceberg, high availability is the default.
+
+
+```sql
+{{ config(
+    materialized='table',
+    ha=true,
+    format='parquet',
+    table_type='hive',
+    partitioned_by=['status'],
+    s3_data_naming='table_unique'
+) }}
+
+select 'a'      as user_id,
+       'pi'     as user_name,
+       'active' as status
+union all
+select 'b'        as user_id,
+       'sh'       as user_name,
+       'disabled' as status
+```
+
+By default, the materialization keeps the last 4 table versions,but you can change it by setting `versions_to_keep`.
```

Comment:
sugg moving this sentence to above the code block. feels like this could get lost in the scanning